lang/python/python/Makefile

```diff
@@ -311,18 +311,32 @@ define Host/Compile
 endef
 
 define Host/Install
+	$(if $(wildcard $(HOST_PYTHON_PKG_DIR)/.setuptools_installed_$(PYTHON_SETUPTOOLS_VERSION)-$(PYTHON_SETUPTOOLS_PKG_RELEASE)),,
+		rm -rf \
+			$(HOST_PYTHON_PKG_DIR)/easy_install.py \
```

Comment:
won't removing these files, mean that they will also be installed ?
.i.e regardless of whether something changed ?

Reply:
Yes, but this would happen only when the version and/or `PKG_RELEASE` has changed, which shouldn’t be too often.

As they can also be patched, I think it would be safer to remove all files completely rather than letting the setuptools/pip install keep older versions around (assuming the install doesn’t overwrite them anyway).

Reply:
ack

thanks for clarifying